### PR TITLE
[Selectable Components: Structured List - Style] Website Doc

### PR DESCRIPTION
**Pull request by Kritvi-bhatia17**
Closes [#15775](https://github.com/carbon-design-system/carbon/issues/15775)

For ref: [Figma file](https://www.figma.com/file/w4QA4bnNZAiySPskA5qZMW/Selectable-Components?type=design&node-id=1087-46210&mode=design&t=AwGw5aVBHVUjjaus-4)

---

### New
**_Color_** & **_Interactive states_** section
> - [x] Added an image for both Transparent and Background structured list (enabled states)
> - [x] Added the color specs for structured list with background as 'optional'

**_Structure_** section
> - [x] Add _Default structure_ & _Selectable structure_ sections with description and images each

> - [x] Add a new section(_Size_ section) with description and images each

---

### Changed

**_Color_**, **_Interactive states_** & **_Structure_** section
> - [x] Updated the table have the correct color tokens

### CODE REVIEW

**Review by tw15egan — APPROVED** [review]
LGTM 👍 ✅

**Review by laurenmrice — CHANGES_REQUESTED** [review]
Looking good Kritvi! Just some comments below: 

**Structure section**
Since you are showing the hang and flush structured list specs in the image, the table above it also needs to have some information for the flush list specs. The table currently only has information for the hang structured list specs.

I would also add specs to the table for the Icon left and right padding.

File: src/pages/components/structured-list/style.mdx [hunk 1]
```diff
@@ -52,34 +126,40 @@ set in sentence case. All typography is left aligned.
...
 </Caption>
 
+## Size
+
+There are two structured list sizes: default and condensed.
+
 <div className="image--fixed">
 
-![Spacing and measurements for structured list with selection](images/structured-list-style-3.png)
+![Sizes of structured list with selection](images/structured-list-style-7.png)
```
Comment (laurenmrice) [inline src/pages/components/structured-list/style.mdx:161]: This image in the Size section could be simplified a bit with the specs, to direct the users attention more to the specifications of what changes between the condensed and default  heights. I will create an example of what I mean in your figma file.

**Review by Kritvi-bhatia17 — COMMENTED** [review]
.

**Comment by laurenmrice** [comment]
Looks good Kritvi! I think after these small image bugs, it should be good to go!

I would check the bottom and top spacing in these areas in red in the following images (for both background and transparent images) to make sure they are the same size and that the component is centered in the center of the space. I can't tell if its the tab component in gatsby that is causing this, or if its in the actual image file.

Screenshot 2024-03-21 at 8 26 39 AM

Screenshot 2024-03-21 at 8 30 57 AM

------

This image just needs to bring this top divider in, its too long.
Screenshot 2024-03-21 at 8 33 12 AM

**Comment by Kritvi-bhatia17** [comment]
> Looks good Kritvi! I think after these small image bugs, it should be good to go!
> 
> I would check the bottom and top spacing in these areas in red in the following images (for both background and transparent images) to make sure they are the same size and that the component is centered in the center of the space. I can't tell if its the tab component in gatsby that is causing this, or if its in the actual image file.
> 
> Screenshot 2024-03-21 at 8 26 39 AM Screenshot 2024-03-21 at 8 30 57 AM
> This image just needs to bring this top divider in, its too long. Screenshot 2024-03-21 at 8 33 12 AM

Hi Lauren!
Oh damn, thank you for checking. Here are a couple of points:

- The preview link isn't showing the latest update for some reason.
- While the images have the corrected spacing, there seems to be an issue with the code introducing a tab above it, causing unequal spacing top and bottom. I'm not sure if it's a code issue, but the images in Figma have the correct spacing. I think I might need to connect with one of the developers to address this.
- Regarding the last image you pointed out, if you check under [files changed](https://github.com/carbon-design-system/carbon-website/pull/3965/files#diff-e8788555405bb6e0491639f3f59e01fc8fef304c0636de505599028c070b2593), it appears to be correct there, but it's not reflecting correctly on the preview link provided.

Thanks again!

**Review by laurenmrice — APPROVED** [review]

**Comment by Kritvi-bhatia17** [comment]
> Looks good Kritvi! I think after these small image bugs, it should be good to go!
> 
> I would check the bottom and top spacing in these areas in red in the following images (for both background and transparent images) to make sure they are the same size and that the component is centered in the center of the space. I can't tell if its the tab component in gatsby that is causing this, or if its in the actual image file.
> 
> Screenshot 2024-03-21 at 8 26 39 AM Screenshot 2024-03-21 at 8 30 57 AM
> This image just needs to bring this top divider in, its too long. Screenshot 2024-03-21 at 8 33 12 AM


Hi @laurenmrice! I spoke with @tay1orjones to clarify a couple of points:
1. Regarding the padding issue, I've raised [an issue](https://github.com/carbon-design-system/carbon-website/issues/3987#issue-2213719797) after discussing with @tay1orjones. There seems to be unequal padding set in the code, and we might need to discuss it with the team.
2. The preview link has been updated, and it now displays the latest images.

The padding issue might take some time to resolve. In the meantime, I'll go ahead and `merge this PR`. Just wanted to keep you in the loop!

**Pull request merged**